README.md

```diff
@@ -36,5 +36,21 @@ book](https://doc.rust-lang.org/book/2018-edition/ch01-01-installation.html)).
 
 See the [crux-mir][crux-mir-repo] README for usage instructions.
 
+## JSON schema
+
+`mir-json` and related tools produce MIR JSON files as output, which the
+contain the intermediate MIR code from the compiled program in a
+machine-readable format. Downstream tools are sensitive to the particular
+schema that a MIR JSON file uses, so we explicitly record the version of the
+JSON schema in each MIR JSON file (in the `"version"` field).
+
+Any time that `mir-json` is updated such that the JSON schema must also be
+changed, we will also update the schema version number. This version number
+follows [Semantic Versioning](https://semver.org/). As such, an update to the
```

Comment:
I wonder how much benefit we'll actually get out of using semver instead of a single number (with all versions assumed to be incompatible).  Most changes to the schema so far have been non-backwards-compatible.

Reply:
Yes, this is a fair point. I don't have any particular examples in mind where having a minor version would be helpful, so I'll change this from a semver to a simple `u64` instead. We can always generalize this later if requiring a major version bump for each change causes too much churn.